lib/interfaces/swagger-custom-options.interface.ts

```diff
@@ -8,6 +8,7 @@ export interface SwaggerCustomOptions {
   swaggerUrl?: string;
   customSiteTitle?: string;
   validatorUrl?: string;
+  uiConfig?: Record<string, any>;
```

Comment:
We should define a separate options object for the fastify platform.

Reply:
With just the uiConfig property (for now)?

Reply:
Sounds good! Since we're about to introduce a new `FastifySwaggerCustomOptions` type, I think we should rename the existing one (`SwaggerCustomOptions`) to `ExpressSwaggerCustomOptions` and define a union `type SwaggerCustomOptions = ExpressSwaggerCustomOptions | FastifySwaggerCustomOptions`

Reply:
@kamilmysliwiec sorry, had a lot on my plate but I've done the split! :)